paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetAddPaymentMethodFragmentTest.kt

```diff
@@ -203,6 +196,7 @@ internal class PaymentSheetAddPaymentMethodFragmentTest : PaymentSheetViewModelT
     }
 
     @Test
+    @Ignore("This test is failing silently on master.")
```

Comment:
It's nice to see we've narrowed this down. Would be nice to figure out the root cause of this.